### PR TITLE
Teacher search page

### PR DESCRIPTION
**Pull request by Rebsonau**

### CODE REVIEW

**Review by rockettdev — DISMISSED** [review]
LGTM!

**Review by Kocibey01 — DISMISSED** [review]

File: src/pages/search/index.js [hunk 1]
```diff
@@ -77,76 +83,134 @@ function SearchPage () {
...
-                        value={formData}
-                        onChange={onChange}
-                        name={'searchusers'}
-                        type="text"
-                        icon={<SearchIcon />}/>
-                        
-                        </form>
-                    )}
-
+            {userRole === 'TEACHER' ? (
```
Comment (dearshrewdwit) [inline src/pages/search/index.js:86]: There seems to be a lot of duplication here.

Could you find a way to do something like:
```
 { 
    userRole === 'Teacher' && (
      <div>Add Note</div>
      <div>Move to Cohort</div>
    )
}
```

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]

**Review by Kocibey01 — DISMISSED** [review]

**Review by Kocibey01 — COMMENTED** [review]
Nice!

**Comment by Kocibey01** [comment]
> Nice!

**Review by dearshrewdwit — APPROVED** [review]

**Review by Kocibey01 — APPROVED** [review]

**Pull request merged**